client/src/components/ResultData.js

```diff
@@ -1,10 +1,15 @@
 export default class ResultData {
-  constructor(title, description, excerpt, location, postUrl, applyUrl) {
-    this.title = title;
+  constructor(job_title, link, description, city, state, country, latitude, longitude, company_name, salary_min, start_date) {
+    this.job_title = job_title;
```

Comment:
I think it makes sense to have id in here, just in case it could be used in the future for sending out requests e.g. `GET/job/:id`.

Reply:
Added id